packages/react/src/xcss-prop/index.ts

```diff
@@ -0,0 +1,143 @@
+import type * as CSS from 'csstype';
+
+import { ac } from '../runtime';
+import type { CSSPseudos } from '../types';
+
+type CSSProperties = Readonly<CSS.PropertiesFallback<number | string>>;
```

Comment:
TIL PropertiesFallback - is providing a CSS value as an array something we support? e.g.

```ts
{ display: ['-webkit-flex', 'flex'], }```

Reply:
I can't remember. Not with these APIs at least. I'd say we don't need fallbacks thanks to autoprefixer. I'll remove the fallbacks for now!